set.mm

```diff
@@ -46129,13 +46114,7 @@ This theorem is proved directly from set theory axioms (no set theory
        16-Jan-2012.)  (Revised by NM, 16-Jun-2017.) $)
     reuxfr2d $p |- ( ph
```

Comment:
This theorem seems to be obsolete now (it's identical with ~reuxfr3d). However, this version is older than ~reuxfr3d, therefore ~reuxfr2d  should be kept, and ~reuxfr3d should be removed. Since the proof of ~reuxfr3d is shorter, this proof should be kept.
In summary:
* Replace the proof of ~reuxfr2d by the proof of ~reuxfr3d and add "(Proof shortened by Thierry Arnoux, 8-Oct-2017.)" to the comment of ~reuxfr2d 
* Replace all usages of ~reuxfr3d  by ~reuxfr2d
* Remove ~reuxfr3d

Reply:
reuxfr3d is referenced by a theorem that appears earlier than reuxfr2d, is it ok if I move reuxfr2d in the same spot where reuxfr3d was placed?

Reply:
yes

Reply:
`reuxfr3d` is not exactly identical to `reuxfr2d` and the proofs are not interchangeable, it's possible to find a shorter proof automatically for `reuxfr2d` (even shorter than the one of `reuxfr3d` by one byte), but `reuxfr3d` is used by `reuxfr4d` and a simple substitution (or authomation tricks) don't work for it. This actually needs manual proof revision, and I think this would deserve its own pull request...

Reply:
it seems like reuxfr3d is a slightly generalized version of reuxfr2d, with y e. C instead of y e. B (C can be substituted with B)
i don't really understand the naming system... so far it seems like "2" just means "2nd reuxfr theorem"
so instead maybe rename reuxfr3d to reuxfr2d, deleting reuxfr2d (subtly different)